cigaradvisor/blocks/search-results/search-results.js

```diff
@@ -93,21 +14,23 @@ function filterData(fullTerm, data) {
  * @param {number} articlesCount - The total count of articles. This is needed for pagination.
  * @returns {Promise<void>} - A promise that resolves when the page is rendered.
  */
-async function processSearchResults(results, allArticles, wrapper, limit, articlesCount) {
-  const articles = [];
-  results.forEach((post) => {
-    const filteredArticles = allArticles.filter((obj) => obj.path === getRelativePath(post.path));
-    articles.push(filteredArticles[0]);
-  });
+async function processSearchResults(articles, wrapper, limit, articlesCount) {
   await renderPage(wrapper, articles, limit, articlesCount);
   const loadingImageContainer = document.querySelector('.loading-image-container');
   const articleListWrapper = document.querySelector('.article-list-wrapper');
+  loadingImageContainer.style.transition = 'opacity 2s';
```

Comment:
set this as a CSS declaration. JS should just add/remove a class to the parent element which will activate these

Reply:
fixed